### ISSUE TITLE
Running npm install gives Error: not supported

### ISSUE THREAD

**Issue by rulloliver** [oguilleux/webpack-gulp-wordpress-starter-theme#760]
Thank you for the Gulp theme! Getting an error when trying to run `npm install`

- Tried deleting node modules for a clean reinstall
- tried deleting package-lock.json for a clean reinstall

npm 6.13.4
node v12.16.1

```
Error: Not supported
    at eval (eval at <anonymous> (..../node_modules/gulp/node_modules/gulp-cli/lib/shared/require-or-import.js:10:15), <anonymous>:1:1)
    at requireOrImport (..../node_modules/gulp/node_modules/gulp-cli/lib/shared/require-or-import.js:24:7)
    at execute (..../node_modules/gulp/node_modules/gulp-cli/lib/versioned/^4.0.0/index.js:37:3)
    at Liftoff.handleArguments (..../node_modules/gulp/node_modules/gulp-cli/index.js:211:24)
    at Liftoff.execute (..../node_modules/liftoff/index.js:201:12)
    at module.exports (..../node_modules/flagged-respawn/index.js:51:3)
    at Liftoff.<anonymous> (..../node_modules/liftoff/index.js:191:5)
    at ..../node_modules/liftoff/index.js:149:9
    at ..../node_modules/v8flags/index.js:162:14
    at ..../node_modules/v8flags/index.js:41:14
```

Any ideas what could be wrong here? Likely seems likely a versioning issue with some package but no idea currently how approach with this error. Any advice?

**Comment by oguilleux** [comment]
Hi @rulloliver, try updating your versions of npm and node. I think it's what causes the errors.

**Comment by oguilleux** [comment]
careful I just saw that support for node v12 [is over.](https://nodejs.org/es/blog/release/v12.22.12/)

**Comment by rulloliver** [comment]
@oguilleux Thanks for this - played around got it working with updated versions. Although regarding warnings & errors (I'm a bit of monk i guess although it works): 
`- Deprecation Warning: $weight: Passing a number without unit % (100) is deprecated.`
* although saw this will be fixed in Bootstrap 5.2.3
```
- ✖ 35 problems (34 errors, 1 warning)
  21 errors and 0 warnings potentially fixable with the `--fix` option.
```
- how did you resolve these? Seems like --fix doesn't resolve any.

Thanks for the quick discussion!

**Comment by oguilleux** [comment]
Yes it's a [known issue](https://github.com/twbs/bootstrap/pull/37425) and it will be fixed in Bootstrap v5.3.0

**Comment by oguilleux** [comment]
I don't see the
```
- ✖ 35 problems (34 errors, 1 warning)
  21 errors and 0 warnings potentially fixable with the `--fix` option.
```
I only have 1 warning when installing the project and it's the console.log I left for demo purposes.
